ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -101,6 +101,9 @@ fi
 if test -f "${SHARED_DIR}/proxy-conf.sh"; then
   if [[ "${CLUSTER_TYPE}" =~ ^aws-s?c2s$ ]]; then
     echo "proxy-conf.sh detected, but not reqquired by C2S/SC2S while destroying cluster, skip proxy setting"
+  elif [[ "${JOB_NAME}" =~ .*azure-ipi-public-to-private.* ]]; then
```

Comment:
I guess we can use `CLUSTER_TYPE` for the case to avoid the job name hardcode.
```suggestion
  elif [[ "${CLUSTER_TYPE}" == "azure4" ]]; then
      # when bastion host is provisioned in cluster resource group, once the bastion is destroyed in the destroy process, the running destroy process would be interrupted and failed. E.g: azure-ipi-public-to-private jobs.
    echo "proxy-conf.sh detected, but not required by azure4 clusters while destroying cluster, skip proxy setting"
```